crates/common/types/genesis.rs

```diff
@@ -198,6 +204,9 @@ pub struct ChainConfig {
     #[rkyv(with = rkyv_utils::H160Wrapper)]
     // Deposits system contract address
     pub deposit_contract_address: Address,
+
+    #[serde(default)]
+    pub enable_verkle_at_genesis: bool,
```

Comment:
is this necessary for the tests to run?

Reply:
verkle is something that is not planned for fusaka, this shouldn't be necessary

Reply:
please exclude those test them. I don't think we should have a reference to verkle in the genesis